common/types/null.go

```diff
@@ -38,16 +40,18 @@ func (n Null) ConvertToNative(typeDesc reflect.Type) (interface{}, error) {
 	switch typeDesc.Kind() {
 	case reflect.Ptr:
 		switch typeDesc {
-		case jsonValueType:
-			return &structpb.Value{
-				Kind: &structpb.Value_NullValue{
-					NullValue: structpb.NullValue_NULL_VALUE}}, nil
 		case anyValueType:
 			pb, err := n.ConvertToNative(jsonValueType)
```

Comment:
Prefer g.p.NullValue instead of g.p.Value.

Reply:
I don't think I can in this case since the NullValue is an enum and not a `proto.Message` instance.